src/middlewares/auth.middleware.ts

```diff
@@ -1,31 +1,46 @@
 import { NextFunction, Response } from 'express';
 import { verify } from 'jsonwebtoken';
-import { SECRET_KEY } from '@/config';
+import { ACCESS_TOKEN_SECRET_KEY, REFRESH_TOKEN_SECRET_KEY } from '@/config';
 import { UserEntity } from '@/entities';
 import { HttpException } from '@/exceptions';
+import { AuthService } from '@/services';
 import { DataStoredInToken, RequestWithUser } from '@/interfaces';
 
-const authMiddleware = async (req: RequestWithUser, _res: Response, next: NextFunction) => {
+const authMiddleware = async (req: RequestWithUser, res: Response, next: NextFunction) => {
+  const refreshToken = req.cookies['RefreshToken'] || null;
```

Comment:
Wouldn't this default to a falsy value if it doesn't exist in the cookie?
If it is, adding null is unneccessary.